libraries/Microsoft.Bot.Builder/TurnStateConstants.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Schema;
+
+namespace Microsoft.Bot.Builder
+{
+    /// <summary>
+    /// Constants used in TurnState.
+    /// </summary>
+    public static class TurnStateConstants
+    {
+        /// <summary>
+        /// TurnState key for the OAuth login timeout.
+        /// </summary>
+        public const string OAuthLoginTimeoutKey = "LoginTimeout";
+
+        /// <summary>
+        /// Default amount of time an OAuthCard will remain active.
+        /// </summary>
+        public const int OAuthLoginTimeoutMsValue = 900000;
```

Comment:
900,000 == 15 minutes. Any reason this value was chosen? Please add a comment.

Reply:
Also, I suggest `public const TimeSpan OAuthLoginTimeout = TimeSpan.FromMinutes(15);`

Reply:
I didn't chose it, Steve did (see, I removed where he put it only in the OAuthPrompt and moved it more centrally). It's just the amount of time the card stays active and has been there forever, but I'll add a comment now and change it to a TimeSpan